DEVELOPMENT.md

```diff
@@ -87,6 +87,23 @@ Maintain the same quality for production and test code.
 
 When writing a Git commit message, follow these [guidelines](https://chris.beams.io/posts/git-commit/).
 
+### Git merge strategy
+
+PRs are usually merged into `master` using the  `rebase and merge` strategy.
```

Comment:
`PR` -> `Pull request`. It is more or less official documentation so let's be crystal clear.